assembl/static2/js/app/graphql/AllLanguagePreferences.graphql

```diff
@@ -0,0 +1,8 @@
+query AllLanguagePreferences($inLocale: String!) {
```

Comment:
I don't understand the difference between `AllLanguagePreferences.graphql` and `DiscussionPerefrenceLanguage.graphql`. Can you explain?

Reply:
They have the exact same shape, hence the confusion. But `AllLanguagePreferences` queries all available languages across the server, regardless of discussion. The `DiscussionPreferenceLanguage` queries only the the available languages of a specific discussion.